File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
      int Height=0;
   do 
    {
    Height= get_int("Height: ");
    }
    while (Height>8 || Height<1);
    for (int i=1; i<=Height; i++){
        for (int j=1; j<=Height-i; j++){
            
            printf(" ");
        }
        for (int k=1; k<=i; k++){
            printf("#");
        }
        printf("  ");
        for (int h=1; h<=i; h++){
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int


height = get_int("Enter height: ")

while (height < 1 or height > 8):
    height = get_int("Enter again: ")

for i in range(1, height+1):
    j = height - i
    while j > 0:
        print(" ", end = "")
        j -= 1
    for t in range(i):
        print("#" , end = "")
    print("  ", end = "")
    print("#" * i)
    
        
